agent/README.md

```diff
@@ -254,6 +254,19 @@ Afterwards, restart Glassfish. Please verify the setup with `asadmin`'s `list-jv
 
 Register the agent by setting a `JAVA_OPTIONS` variable such that the Jetty process can see it.
 
+## Additional steps for SAP NetWeaver Java
+
+Please note that the Teamscale JaCoCo Agent requires at least Java 8, which is part of NetWeaver Java 7.50.
+Prior versions of NetWeaver Java are not supported by the Teamscale JaCoCo Agent.
+
+In order to set the JVM agent parameter, you need to use the NetWeaver Administrator to navigate to
+`Configuration` - `Infrastructure` - `Java System Properties` and add an additional JVM parameter.
+Use the search field to search for `agent`, and select the `-javaagent:<jarpath>[=<options>]` entry.
+In the `name` field, enter the first part, until the `.jar`, and provide all the options (*without* the
```

Comment:
```suggestion
In the `name` field, enter the first part, until (including) the `.jar`, and provide all the options (*without* the leading
```

Probably everyone knows what they are doing here, but I find the until slightly misleading